admin/create-dumps.sh

```diff
@@ -89,6 +90,14 @@ else
     exit
 fi
 
+# Lock cron, so it cannot be accidentally terminated.
+admin/cron_lock.py lock-cron create-dumps "Creating $DUMP_TYPE dump."
```

Comment:
In the previous PR we decided to prefix these with `python [file.py]`, I see it's still there in the python code, so I guess this is an oversight

Reply:
Ah, yes, I only fixed the ones from the python context, not the shell context.